### PR TITLE
ZCS-4965 Add ant tasks for execution of performance tests

### PR DESCRIPTION
**Pull request by msmcgillis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,34 +16,86 @@ This contains zimbra jmeter load tests.
...
+### Execution
 
 ```
-# grab a copy of this repo
 $ get clone https://github.com/Zimbra/zm-load-testing.git
 $ cd zm-load-testing
+$ ant
+```
+
+To run a specific test use that path name of the tests seperated by hypen example:
```
Comment (fatkudu) [inline README.md]: Typo - s/hypen/hyphen/

**Comment by msmcgillis** [comment]
For item 1 we still want the documentation that talks about using -Denv=config/myenv.prop to work so I made the change in how we set env and now use value rather than location so we don't get basedir appended in the default case.

For item 2 I create a logs directory now if it doesn't exist and generate files inside like:

test-name-jmeter.log
test-name-reqeusts.log 

Let me know if you see anything else.

**Comment by sakchhi-kiran** [comment]
I have one more review for these this.
While executing the tests I have observed few exceptions of "(No such file or directory)" in jmeter.log for generic-lmtp, generic-mix and generic-smtp.
2018/05/09 08:59:39 INFO  - jmeter.util.BeanShellTestElement: LMTP User 1-1 DATA: java.io.FileNotFoundException: tests/generic/lmtp/profiles/message.txt (No such file or directory)

This issue gets resolved once I provide complete path for message.txt in "tests/generic/mix/profiles/basic.prop".

**Comment by msmcgillis** [comment]
Fixed the ant code so it sets the paths for those tests appropriately for jmeter ant.

The basic issue around these items is that the jmeter ant module changes the current working directory from basedir to jmeterhome in my view this is a bug in the jmeter code. We could investigate if new versions of jmeter have this same issue and if they do it might be worth opening a issue against jmeter. If this issue is fixed in current jmeter this would be another reason for us to consider working through ZCS-4966.

**Pull request merged**